mojo/core/broker_castanets.cc

```diff
@@ -95,7 +95,6 @@ BrokerCastanets::BrokerCastanets(PlatformHandle handle,
   flags = fcntl(fd, F_SETFL, flags & ~O_NONBLOCK);
   PCHECK(flags != -1);
 
-
   // Wait for the first message, which should contain a handle.
```

Comment:
Could you squash this commit into previous one?

Reply:
done